bitnami/postgresql/templates/secrets.yaml

```diff
@@ -14,16 +22,62 @@ metadata:
 type: Opaque
 data:
   {{- if .Values.auth.enablePostgresUser }}
-  postgres-password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.names.fullname" .) "key" "postgres-password" "providedValues" (list "global.postgresql.auth.postgresPassword" "auth.postgresPassword") "context" $) }}
+  postgres-password: {{ $postgresPassword | b64enc | quote }}
   {{- end }}
   {{- if not (empty (include "postgresql.username" .)) }}
-  password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.names.fullname" .) "key" "password" "providedValues" (list "global.postgresql.auth.password" "auth.password") "context" $) }}
+  password: {{ $password | b64enc | quote }}
   {{- end }}
   {{- if eq .Values.architecture "replication" }}
-  replication-password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.names.fullname" .) "key" "replication-password" "providedValues" (list "auth.replicationPassword") "context" $) }}
+  replication-password: {{ $replicationPassword | b64enc | quote }}
   {{- end }}
   # We don't auto-generate LDAP password when it's not provided as we do for other passwords
   {{- if and .Values.ldap.enabled (or .Values.ldap.bind_password .Values.ldap.bindpw) }}
-  ldap-password: {{ coalesce .Values.ldap.bind_password .Values.ldap.bindpw  | b64enc | quote }}
+  ldap-password: {{ $ldapPassword  | b64enc | quote }}
   {{- end }}
-{{- end -}}
+{{- if .Values.serviceBindings.enabled }}
```

Comment:
Does it make sense to have this`if` condition outside of the global createSecret if. That way at least in some scenarios there would only be one secret.

Reply:
Not sure if I follow. There will be only one secret when `serviceBindings.enabled=false`, which will be the default secret. When this flag is set to true, we will have the default secret plus the serviceBindings one.